README.md

```diff
@@ -1,3 +1,226 @@
-# kessel-sdk-py
+# Kessel SDK for Python
+
+A Python SDK for connecting to Kessel services using gRPC.
+
+## Table of Contents
+
+- [Installation](#installation)
+- [Quick Start](#quick-start)
+- [Client Builder](#client-builder)
+- [Configuration](#configuration)
+  - [Target Server](#target-server)
+  - [Credentials](#credentials)
+  - [Keep-Alive Settings](#keep-alive-settings)
+- [Authentication](#authentication)
+- [Examples](#examples)
+- [API Reference](#api-reference)
+- [Error Handling](#error-handling)
+- [Development](#development)
+
+
+## Installation
+
+```bash
+pip install kessel-sdk
+```
+
+## Quick Start
+
+```python
+import grpc
+
+from kessel.inventory.v1beta2 import (
+    ClientBuilder, 
+    rbac, 
+    check_request_pb2,
+    resource_reference_pb2, 
+    reporter_reference_pb2
+)
+
+# Build a stub pointing at a local instance
+stub = (
+    ClientBuilder
+    .with_defaults_localhost(9000)
+    .build_inventory_stub()
```

Comment:
We could probably reduce complexity by just doing `build` - In the contexf of the `ClientBuilder` it makes sense that builds gives us a full featured client where we can make calls. 

I recall it had other methods to build the channel. I see the channel as something intermediary that might be less used as we want to reuse the stub as well anyway.